presets/ragengine/models.py

```diff
@@ -22,7 +22,7 @@ class QueryRequest(BaseModel):
     index_name: str
     query: str
     top_k: int = 10
-    llm_params: Optional[Dict] = None  # Accept a dictionary for parameters
+    model_config = ConfigDict(extra='allow')
```

Comment:
Why do you make this change?

Reply:
this change breaks using kaito inference endpoint (llm_params)